nitric-hono/README.md

```diff
@@ -0,0 +1,71 @@
+<p align="center">
+  <a href="https://nitric.io">
+    <img src="https://raw.githubusercontent.com/nitrictech/nitric/main/docs/assets/nitric-logo.svg" width="120" alt="Nitric Logo"/>
+  </a>
+</p>
+
+<p align="center">
+  A fast & fun way to build portable cloud-native applications
+</p>
+
+<p align="center">
+  <img alt="GitHub release (latest SemVer)" src="https://img.shields.io/github/v/release/nitrictech/nitric?sort=semver">
+  <a href="https://twitter.com/nitric_io">
+    <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/nitric_io?label=Follow&style=social">
+  </a>
+  <a href="https://discord.gg/Webemece5C"><img alt="Discord" src="https://img.shields.io/discord/955259353043173427?label=discord"></a>
+</p>
+
+## Project Description
+
+Simple & secure direct-to-bucket (e.g. S3) file uploads from modern browsers with signed URLs using the [Hono Web Framework](https://hono.dev/) for APIs and the Nitric SDK for bucket and file access.
+
+## Usage
+
+### Step 1: Install Nitric
+
+Follow the steps in the [installation guide](https://nitric.io/docs/installation)
+
+### Step 2: Run your project locally Nitric
```

Comment:
```suggestion
### Step 2: Run your Nitric project locally
```

Reply:
@HomelessDinosaur Copied the README from another example. Will apply these to every example readme in another commit.